.pre-commit-config.yaml

```diff
@@ -51,7 +51,7 @@ repos:
         types: [python]
       - id: ruff
         name: ruff
-        entry: ruff check --fix --exit-non-zero-on-fix
+        entry: ruff --fix --exit-non-zero-on-fix
```

Comment:
Keep the check-option. Otherwise you get warnings.

Reply:
fixed in latest commit